result1.csv

```diff
@@ -0,0 +1,4 @@
+fruit,quantity
+banana,43
+orange,112
+apple,39
```

Comment:
All these files ought to be in "resources" directory

Reply:
done